1.请说出throw的使用位置：
    在方法内部使用，根据条件判断，手动抛出异常
2.请说出throws的位置，作用是什么？
    在方法名后进行异常类的声明，但是该异常不会被处理，而是提醒给调用者去处理